src/Enums.js

```diff
@@ -0,0 +1,31 @@
+export const membershipStatus = [
```

Comment:
you can also do
```js
export const membershipStatus = {
  BAN: -2,
  PENDING: -1,
   // ...
}
```

that is likely better than string to int mapping as you can use the `membershipStatus` variables directly

Reply:
the problem with that is idk how to get the key from value